packages/react-component-library/playwright.config.ts

```diff
@@ -0,0 +1,123 @@
+import { PlaywrightTestConfig, devices } from '@playwright/test'
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  testDir: './playwright/tests',
+
+  globalSetup: require.resolve('./playwright/tests/globalSetup'),
+
+  /* Maximum time one test can run for. */
+  timeout: 30 * 1000,
+
+  expect: {
+    /**
+     * Maximum time expect() should wait for the condition to be met.
+     * For example in `await expect(locator).toHaveText();`
+     */
+    timeout: 5000,
+  },
+
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+
+  maxFailures: process.env.CI ? 10 : undefined,
+
+  /* Set to 1 locally to enable trace recording. */
+  retries: process.env.CI ? 1 : 0,
+
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: [
+    [process.env.CI ? 'github' : 'line'],
+    [
+      'html',
+      {
+        outputFolder: './playwright/test-results/html/',
+        open: process.env.PWDEBUG ? 'never' : 'on-failure',
```

Comment:
By default it opens the HTML report on test failure. I tweaked it to not open it if the debugger is being.